src/main/java/org/apache/accumulo/testing/continuous/ContinuousScanner.java

```diff
@@ -62,8 +62,11 @@ public static void main(String[] args) throws Exception {
 
           long t1 = System.currentTimeMillis();
 
-          long count = scanner.stream()
-              .peek(entry -> ContinuousWalk.validate(entry.getKey(), entry.getValue())).count();
+          long count = 0;
+          for (var entry : scanner) {
+            ContinuousWalk.validate(entry.getKey(), entry.getValue());
+            count++;
+          }
```

Comment:
My IDE was warning me that the `.peek` operation could possibly be optimized out by the compiler so I changed the syntax so that wouldnt happen